sync_diff_inspector/config/config.go

```diff
@@ -108,12 +108,13 @@ type Security struct {
 
 // DataSource represents the Source Config.
 type DataSource struct {
-	Host     string             `toml:"host" json:"host"`
-	Port     int                `toml:"port" json:"port"`
-	User     string             `toml:"user" json:"user"`
-	Password utils.SecretString `toml:"password" json:"password"`
-	SqlMode  string             `toml:"sql-mode" json:"sql-mode"`
-	Snapshot string             `toml:"snapshot" json:"snapshot"`
+	Host             string             `toml:"host" json:"host"`
+	Port             int                `toml:"port" json:"port"`
+	User             string             `toml:"user" json:"user"`
+	Password         utils.SecretString `toml:"password" json:"password"`
+	SqlMode          string             `toml:"sql-mode" json:"sql-mode"`
+	Snapshot         string             `toml:"snapshot" json:"snapshot"`
+	MaxExecutionTime int                `toml:"max-execution-time" json:"max-execution-time,omitempty"`
```

Comment:
we are planning to make it accept all session variables ([similar to `target-database.session` in DM](https://docs.pingcap.com/tidb/stable/task-configuration-file-full)), so it can support `max_execution_time` as well as other options e.g. [to tune the optimizer](https://docs.pingcap.com/tidb/stable/system-variables#tidb_opt_agg_push_down) etc.

Reply:
@kennytm Ok thank you. I'll follow that PR instead.

I'll close this PR